### PR TITLE
Add Parquet/Feather blog 1/3

### PR DESCRIPTION
**Pull request by mavam**
This PR adds the fist blog post from our upcoming Parquet & Feather series.

### :memo: Reviewer Checklist

Build locally and review while reading.

### CODE REVIEW

**Review by dispanser — APPROVED** [review]
Great read - I added a few minor spelling improvement proposals (SIP), and a note about how it's not _that_ easy to just point an analytics engine to our storage.

While reading this, I got the impression that these blog posts also serve as a nice introductory text for the VAST architecture - seeing the image with the catalog, I would have loved to add a link to a blog post about the catalog, and from there to our indexing ... maybe this could be the start for a more thorough VAST architecture blog post series!

File: web/blog/parquet-and-feather-1-of-3-enabling-open-investigations/index.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+monoliths fail: they offer a single black box that tight couple data acquision
+and processing capabilities. Providing a thin "open" API is not really open, as
+it prevents high-bandwidth data access that is need for advanced analytics
+workloads.
+
+Open storage prevents vendor-lock-in. When any tool can work with the data, you
+build a sustainable foundation for implementing future use cases. For example,
+with Parquet's column encryption, you can offload fine-grained compliance use
+cases to a dedicated application. Want to try out a new analytics engine? Just
+point it to the Parquet files.
```
[the remark this answers is not in the document]
Comment (mavam) [reply web/blog/parquet-and-feather-1-of-3-enabling-open-investigations/index.md]: We agreed that this is fine.

File: web/blog/parquet-and-feather-enabling-open-investigations/index.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+---
+
+[Apache Parquet][parquet] is the common denominator for structured data at rest.
+The data science ecosystem has long appreciated this. But infosec? Why should
+you care about Parquet when building a threat detection and investigation
+platform? In this blog post series we share our opinionated view on this
+question. In the next three blog posts, we
+
+1. describe how VAST uses Parquet and its little brother [Feather][feather]
+2. benchmark the two foramts against each other for typical workloads
```
Comment (dispanser) [inline web/blog/parquet-and-feather-enabling-open-investigations/index.md:17]:
```suggestion
2. benchmark the two formats against each other for typical workloads
```

File: web/blog/parquet-and-feather-enabling-open-investigations/index.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+memory mapping and zero-copy usage. On the spectrum of speed and
+space-efficiency, think of it this way:
+
+![Parquet vs. Feather](parquet-vs-feather.light.png#gh-light-mode-only)
+![Parquet vs. Feather](parquet-vs-feather.dark.png#gh-dark-mode-only)
+
+Before Feather existed, VAST had its own storage format that was 95% like
+Feather, minus a thin framing. (We called it the *segment store*.)
+
+Wait, but Feather is an in-memory format an Parquet an on-disk format. You
```
Comment (dispanser) [inline web/blog/parquet-and-feather-enabling-open-investigations/index.md:69]:
```suggestion
Wait, but Feather is an in-memory format and Parquet an on-disk format. You
```

File: web/blog/parquet-and-feather-enabling-open-investigations/index.md [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+batches to disk. The diagram below illustrates the architecture:
+
+![Parquet Analytics](parquet-analytics.light.png#gh-light-mode-only)
+![Parquet Analytics](parquet-analytics.dark.png#gh-dark-mode-only)
+
+[arrow-table]: https://arrow.apache.org/docs/python/data.html#tables
+[arrow-record-batch]: https://arrow.apache.org/docs/python/data.html#record-batches
+[store-plugin]: /docs/understand/architecture/plugins#store
+
+This architecture makes it easy to point analytics application directly to the
```
Comment (dispanser) [inline web/blog/parquet-and-feather-enabling-open-investigations/index.md:94]:
```suggestion
This architecture makes it easy to point an analytics application directly to the
```

**Review by dispanser — APPROVED** [review]
The flow got smoother on reading, I think it's a great improvement. Found three instances of what I think might mispelled words.

**Pull request merged**